src/app/space/create/deployments/services/deployments.service.spec.ts

```diff
@@ -1290,9 +1292,11 @@ describe('DeploymentsService', () => {
       const expectedResponse = {
         total: 6,
```

Comment:
I guess pod_total and the actual total aren't connected in any way since the backend supplies the values. Would be nice to have it correct in the tests; though it's a trivial nit.

Reply:
Whoops - yea, good catch. I'll update it so the test acts as a better reference for what a real response is expected to look like.